reference/04-Patterns/overview.md

```diff
@@ -281,12 +282,12 @@ There is one item that you need to be aware about in regards to Block Patterns.
 
```

Comment:
It may be nice to link to the synced patterns document from here

Reply:
Done!